src/helm/benchmark/scenarios/bhasa_scenario.py

```diff
@@ -95,8 +95,8 @@ def get_instances(self, output_path) -> List[Instance]:
                 # Select only bottom 20th percentile by length for in-context examples as examples are very long
                 data = df[df["passage_text"].apply(len) < df["passage_text"].apply(len).quantile(.2)]
             else:
-                # Sample 100 examples for test
-                data = df.sample(n=100, random_state=5018)
+                # Sample 565 examples for test
+                data = df.sample(n=565, random_state=5018)
```

Comment:
No need to sample since we are taking the entire validation set as the test set!